srcpkgs/k9s/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'k9s'
 pkgname=k9s
-version=0.25.8
+version=0.25.12
```

Comment:
.14 is out

Reply:
thank you for noticing @ericonr, those guys are fast, I've updated the PR

Reply:
Right? So many releases!

Reply:
Also lmao .15 is now out.

Reply:
They don't sleep for sure. I've updated the PR.